### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by bmobito**
Minor edits to title, contributors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -253,9 +253,9 @@ <h2>History of <strong class="theme_blue">handheld</strong></h2>
                         <div class="col-lg-12 col-md-12 col-sm-12">
                             <h3>Team member</h3>
                             <p>
-                                Daniel Gebreselassie &nbsp; &nbsp; &nbsp; &nbsp; Donnovin Rogers &nbsp; &nbsp; &nbsp; &nbsp; e Brink &nbsp; &nbsp; &nbsp; &nbsp; Jeremy Robinson
+                                Daniel Gebreselassie &nbsp; &nbsp; &nbsp; &nbsp; Donnovin Rogers &nbsp; &nbsp; &nbsp; &nbsp; Emily Brink &nbsp; &nbsp; &nbsp; &nbsp; Jeremy Robinson
```
Comment (BeartoothHiker) [inline index.html:256]: @brinkbrink how would you like your name here?

**Review by BeartoothHiker — COMMENTED** [review]
Minor comments.

File: index.html (hunk 1 above)
Comment (BeartoothHiker) [reply index.html:256]: Never mind on this.  Probably not a big deal.

Comment (brinkbrink) [reply index.html:256]: E. Brink, I'll change it

**Pull request merged**